novawallet/Modules/Notifications/GovernanceNotifications/GovernanceNotificationsViewController.swift

```diff
@@ -0,0 +1,123 @@
+
+    init(
+        presenter: GovernanceNotificationsPresenterProtocol,
+        localizationManager: LocalizationManagerProtocol
+    ) {
+        self.presenter = presenter
+        super.init(
+            presenter: presenter,
+            localizationManager: localizationManager
+        )
+    }
+
+    @available(*, unavailable)
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        presenter.proceed()
+    }
+
+    private func createSections() -> [ChainNotificationSettingsViewController.Section] {
+        viewModels.map(createSection)
+    }
+
+    private func createSection(from model: GovernanceNotificationsModel) -> Section {
+        let newRefendumTitle = R.string.localizable.notificationsManagementGovNewReferendum(
+            preferredLanguages: selectedLocale.rLanguages
+        )
+        let referendumUpdate = R.string.localizable.notificationsManagementGovReferendumUpdate(
+            preferredLanguages: selectedLocale.rLanguages
+        )
+        let delegateHasVoted = R.string.localizable.notificationsManagementGovDelegateHasVoted(
+            preferredLanguages: selectedLocale.rLanguages
+        )
+        let tracks = R.string.localizable.notificationsManagementGovTracks(
+            preferredLanguages: selectedLocale.rLanguages
+        )
+        return .collapsable([
+            .switchCell(.init(
+                title: model.name,
+                icon: model.icon,
+                isOn: model.enabled,
+                action: { self.presenter.changeSettings(chainId: model.identifier, isEnabled: $0) }
+            )),
+            .switchCell(.init(
+                title: newRefendumTitle,
+                icon: nil,
+                isOn: model.newReferendum,
+                action: { self.presenter.changeSettings(
+                    chainId: model.identifier,
+                    newReferendum: $0
+                ) }
+            )),
+            .switchCell(.init(
+                title: referendumUpdate,
+                icon: nil,
+                isOn: model.referendumUpdate,
+                action: { self.presenter.changeSettings(
+                    chainId: model.identifier,
+                    referendumUpdate: $0
+                ) }
+            )),
+            .switchCell(.init(
+                title: delegateHasVoted,
+                icon: nil,
+                isOn: model.delegateHasVoted,
+                action: { self.presenter.changeSettings(
+                    chainId: model.identifier,
+                    delegateHasVoted: $0
+                ) }
+            )),
+            .accessoryCell(.init(
+                title: tracks,
+                accessory: tracksSubtitle(from: model.tracks),
+                action: { self.presenter.selectTracks(chainId: model.identifier) }
```

Comment:
I think we are having circular dependency here because self is inherited from ChainsNotificationsViewControllers which has reference to models.